### PR TITLE
Release/v0.10.0

### PR DESCRIPTION
**Pull request by alliesaizan**
## Tests

- [x] no new tests required
- [ ] new tests added
- [ ] existing tests adjusted

## Documentation

- [x] no documentation changes needed
- [ ] user guide added or updated
- [ ] API docs added or updated
- [ ] example notebook added or updated

### CODE REVIEW

**Review by riedgar-ms — APPROVED** [review]

File: fairlearn/__init__.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 from .show_versions import show_versions  # noqa: F401
 
 __name__ = "fairlearn"
-__version__ = "0.10.0.dev0"
+__version__ = "0.10.0"
```
Comment (riedgar-ms) [inline fairlearn/__init__.py:13]: Although.... shouldn't `main` be going to 0.11.0.dev0?

**Review by riedgar-ms — CHANGES_REQUESTED** [review]
Sorry, just noticed the branch

**Comment by riedgar-ms** [comment]
@alliesaizan I don't think this is part of [the Release process](https://fairlearn.org/v0.9/contributor_guide/release.html) is it?

**Comment by alliesaizan** [comment]
> @alliesaizan I don't think this is part of [the Release process](https://fairlearn.org/v0.9/contributor_guide/release.html) is it?

It looks like it's step 4, but let me know if I did something wrong: 
image

**Comment by riedgar-ms** [comment]
> > @alliesaizan I don't think this is part of [the Release process](https://fairlearn.org/v0.9/contributor_guide/release.html) is it?
> 
> It looks like it's step 4, but let me know if I did something wrong: image

Ahhh, that's supposed to be a PR which will merge onto the release branch to update the version. This PR is merging onto `main`

**Comment by alliesaizan** [comment]
> > > @alliesaizan I don't think this is part of [the Release process](https://fairlearn.org/v0.9/contributor_guide/release.html) is it?
> > 
> > 
> > It looks like it's step 4, but let me know if I did something wrong: image
> 
> Ahhh, that's supposed to be a PR which will merge onto the release branch to update the version. This PR is merging onto `main`

Oh ok, so it looks like this is part of the release branch, so I should be able to close this PR and proceed with the remaining steps?

**Comment by riedgar-ms** [comment]
> > > > @alliesaizan I don't think this is part of [the Release process](https://fairlearn.org/v0.9/contributor_guide/release.html) is it?
> > > 
> > > 
> > > It looks like it's step 4, but let me know if I did something wrong: image
> > 
> > 
> > Ahhh, that's supposed to be a PR which will merge onto the release branch to update the version. This PR is merging onto `main`
> 
> Oh ok, so it looks like this is part of the release branch, so I should be able to close this PR and proceed with the remaining steps?

I think so

**Pull request closed without merging**